ddtrace/contrib/aws_lambda/cold_start.py

```diff
@@ -0,0 +1,18 @@
+__cold_start = True
```

Comment:
We're trying to avoid global variables from the package `datadog-lambda` which have the name: `_cold_start`, in order to avoid the conflict, I've decided to add an extra underscore. Would it be better to rename this variable to be something like `_ddtrace_cold_start`?